src/lib/webhook.ts

```diff
@@ -0,0 +1,19 @@
+import axios from "axios"
+import * as process from "process"
+
+export async function sendWebhook<T>(
```

Comment:
Could add a quick ChatGPT generated ts doc comment above this method, since this is a general purpose method 😁